### DISCUSSION TITLE
Hide the Github Changelog on the home screen

### DISCUSSION THREAD

**Discussion by alabhyajindal** [community/community#16544] [Other Feature Feedback, Questions, & Ideas]
Is there a way to hide this? 
<img width="218" alt="image" src="https://user-images.githubusercontent.com/52493077/168340047-fcb8282c-1cc3-49de-bc05-1041b60699af.png">

**Comment by victor-gp** [comment]
Or at least place the changelog below "Explore repositories"? I used that ~~to procrastinate~~ to discover interesting projects...
